index.html

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
```

Comment:
- [ ] Please add a professional Readme file and remember to use Microverse template